0chromium-imx-aarch64/Dockerfile

```diff
@@ -0,0 +1,139 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# For testing in the LmP
+# docker build --tag hub.foundries.io/lmp/0chromium-imx-aarch64:latest .
+
+FROM debian:bullseye-20220316-slim AS build
+
+LABEL maintainer="Raul Munoz <raul@foundries.io>"
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt-get update -y && \
+    apt-get install -y vim curl wget \
+                       libwayland-client0 libwayland-server0 \
+                       xcompmgr x11-utils uuid-dev subversion ruby \
+                       rpm python-setuptools \
+                       python-dev \
+                       python pkg-config p7zip openbox mesa-common-dev \
+                       libxtst6 libxtst-dev libxt-dev libxss-dev libxslt1-dev \
+                       libxshmfence-dev libxrender1 libxrandr2 libxkbcommon-dev \
+                       libxinerama1 libxi6 libxfixes3 libxdamage1 libxcursor1 \
+                       libxcomposite1 libx11-xcb1 libwww-perl \
+                       libwayland-egl1-mesa libvulkan1 libvulkan-dev libva-dev \
+                       libudev-dev libssl-dev libsqlite3-dev libspeechd2 \
+                       libspeechd-dev libsctp-dev libpulse0 libpulse-dev \
+                       libpng16-16 libpixman-1-0 libpci3 libpci-dev \
+                       libpango-1.0-0 libnss3-dev libnss3 libnspr4-dev \
+                       libnspr4 libkrb5-dev libjpeg-dev libinput10 libinput-dev \
+                       libgtk-3-dev libgtk-3-0 libglu1-mesa-dev libglib2.0-dev \
+                       libglib2.0-0 libgbm1 libgbm-dev libfreetype6 \
+                       libfontconfig1 libffi-dev libevdev2 libevdev-dev \
+                       libelf-dev libdrm2 libdrm-dev libcurl4-gnutls-dev \
+                       libcups2-dev libcups2 libcap2 libcap-dev libcairo2-dev \
+                       libcairo2 libc6-dev libbz2-dev libbrlapi-dev \
+                       libbluetooth-dev libatspi2.0-dev libatspi2.0-0 \
+                       libatk1.0-0 libasound2-dev libasound2 \
+                       gperf flex \
+                       fakeroot elfutils devscripts dbus-x11 cdbs bison \
+                       binutils-arm-linux-gnueabihf binutils-aarch64-linux-gnu \
+                       apache2-bin libflac-dev flac lsb-release wget \
+                       software-properties-common dirmngr apt-transport-https ca-certificates \
+    && apt-get clean && apt-get autoremove && rm -rf /var/lib/apt/lists/*
```

Comment:
All these build tools leak into the final container image. If you move the copy lines below to be done before this, you can do one big RUN command that installs deps, runs the install scripts, and removes the build-deps in one layer making the resulting image a lot smaller

Reply:
Take a look at something like [docker history](https://howchoo.com/docker/how-to-show-all-layers-of-a-docker-image) to inspect the layers of the image you are building. Or use the [dive tool](https://github.com/wagoodman/dive). It will give you some insight

Reply:
hi @doanac, just finished all the optimization on Dockerfile that I found.

Please, take a look again and see what you think.

Raul.